adaptto-2018-example.yaml

```diff
@@ -2,21 +2,24 @@ apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   labels:
-    run: example-kubernetes
-  name: example-kubernetes
+    app: adaptto-2018-example
```

Comment:
Makes sense. I think if we scope the example for adaptto it makes sense that I also rename the repo.

Reply:
+1, was gonna get to that proposal :)